package/android/src/newarch/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -40,10 +40,22 @@ protected ViewManagerDelegate<ReactSlider> getDelegate() {
           new SeekBar.OnSeekBarChangeListener() {
             @Override
             public void onProgressChanged(SeekBar seekbar, int progress, boolean fromUser) {
+              ReactSlider slider = (ReactSlider)seekbar;
+
+              if(progress < slider.getLowerLimit()) {
+                progress = slider.getLowerLimit();
+                seekbar.setProgress(progress);
+              }
+
+              if(progress > slider.getUpperLimit()) {
+                progress = slider.getUpperLimit();
+                seekbar.setProgress(progress);
+              }
```

Comment:
The most important change for Android. It doesn't allow the seekbar to advance if the value is not within the limits